utilities/update_version.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# This script updates the READMEs with the latest non-SNAPSHOT version number and increments the SNAPSHOT 
+# version number in the pom.xml files.
+
+# Precondition: the pom.xml is at a "-SNAPSHOT" version before running this script.  
```

Comment:
This could be invoked by Travis when non snapshot version is detected and after site generation, no?